integration/README.md

```diff
@@ -0,0 +1,54 @@
+# Integration Tests
+
+This tests core use cases for u-root such as:
+
+- retrieving and kexec'ing a Linux kernel,
+- uinit (user init), and
+- running unit tests requiring root privileges.
+
+## Usage
+
+Run the tests with:
+
+    go test
+
+When the QEMU arch is not amd64, set the `UROOT_TESTARCH` variable. For
+example:
+    UROOT_TESTARCH=arm go test
+
+Currently, only amd64 and arm are supported.
+
+## Steps to get the kernel (bzImage)
+#Step 1: clone the open source linux directory:
+git clone https://github.com/torvalds/linux.git
+
+#Step 2: enter the linux directory and see the directories and files within it
+cd linux
+ls
+
+#Step 3: checkout the latest Linux version 
+git checkout v4.17
+
+#Step 4: preload a .config file. You need virtio_pci and virtio_net 
+wget https://raw.githubusercontent.com/u-root/u-root/master/.circleci/images/test-image-amd64/config_linux4.17_x86_64.txt
+
+#Step 5: build the kernel 
+Make -j12 
+
+#Step 6: visit your .bashrc and export the kernel and QEMU 
+Vi ~/.bashrc
+Export the following: 
+#NOTE: THIS IS JUST AN EXAMPLE! Make sure you put the right path to the kernel and QEMU by finding where they are located in your local machine.
+Example: 
+export UROOT_QEMU=”$HOME/bin/qemu-system-x86_64”
```

Comment:
You don't have to 'visit the rc and add this stuff'. That's up to the user. You can just say 
export UROOT_KERNEL="$HOME/linux/arch/x86_64/boot/bzImage"

And why do you assume they have a qemu in $HOME/bin? Almost nobody has that.

Reply:
Ours is in our /usr/, so we just made the changes by copying and pasting our export statements.

Reply:
Ours is in our /usr/. We realize that no body would likely have it stored in their $HOME/bin. So we then made the changes and used the path where ours is stored (/usr/ ......) as an example for the readers.